styles/components/Navbar.scss

```
@@ -3,199 +3,240 @@
+          left: 0;
+          position: absolute;
+          transition: all 0.5s ease-in-out;
+          width: 100%;
+        }
+
+        &::before {
+          transform: translateY(8px);
+        }
+
+        &::after {
+          transform: translateY(-8px);
+        }
+      }
 
-    ul.nav-items li:hover {
-      color: #999;
-      cursor: pointer;
+      &.active {
+        .bar {
+          transform: translateX(-50px);
+        }
 
+        .bar::before {
+          transform: rotate(-45deg) translate(35px, 35px);
+        }
 
-    }
+        .bar::after {
+          transform: rotate(45deg) translate(35px, -35px);
+        }
+      }
 
-    @media (max-height: 800px) {
-      ul.nav-items li {
-        margin: 20px auto;
+      &:hover {
+        .bar,
+        .bar::before,
+        .bar::after {
+          background-color: $acm-cobalt;
+        }
       }
     }
-    @media (max-height: 600px) {
-      ul.nav-items li {
-        margin: 5px auto;
+
+    .nav-items {
+      align-items: center;
+      display: flex; // will become none at smaller screen size
+      list-style: none;
+      margin: 0;
+      padding: 0;
+
+      li {
+        line-height: 40px;
+        padding: 0 20px;
+
+        button {
+          background-color: transparent;
+          border: 0;
+          color: inherit;
+          font-size: 0.95em;
+          font-weight: 600;
+          text-transform: lowercase;
+          transition: 0.2s;
+
+          &.active {
+            color: $acm-cobalt;
+          }
+          
+          &:hover {
+            color: $acm-cobalt;
+            cursor: pointer;
+          }
+        }
+
+        // prevents committee text fade on li hover
+        &:hover {
+          color: $font-primary-color;
+        }
       }
-    }
-    @media (max-height: 400px) {
-      ul.nav-items li {
-        margin: 1px auto;
+
+      #committee-mobile-nav-item {
+        display: none;
+        height: max-content;
+        text-align: center;
+        width: 60%;
+
+        li {
+          padding: 0;
+        }
+
+        // committee color applied on hover and active
+        [href='/committees#ai']:hover,
+        [href='/committees#ai'].active {
+          color: #1EBDF4;
+        }
+
+        [href='/committees#cyber']:hover,
+        [href='/committees#cyber'].active {
+          color: #FFBA44;
+        }
+
+        [href='/committees#design']:hover,
+        [href='/committees#design'].active {
+          color: #FE823C;
+        }
+
+        [href='/committees#hack']:hover,
+        [href='/committees#hack'].active {
+          color: #C960FF;
+        }
+
+        [href='/committees#icpc']:hover,
+        [href='/committees#icpc'].active {
+          color: #FF6B6B;
+        }
+
+        [href='/committees#studio']:hover,
+        [href='/committees#studio'].active {
+          color: #ED3266;
+        }
+
+        [href='/committees#teachLA']:hover,
+        [href='/committees#teachLA'].active {
```

Small bug here due to a typo. 

```suggestion
        [href='/committees#teachla']:hover,
        [href='/committees#teachla'].active {
```